docs/README.md

```diff
@@ -161,96 +148,6 @@ useful.
-| .travis.pre-commit.yml                    | [pre-commit](.github/actions/pre-commit)                                      |
-| .travis.rancher_cli_config.yml            | [setup-rancher-cli](.github/actions/setup-rancher-cli/action.yml)             |
-| .travis.rancher_cli_install.yml           | [setup-rancher-cli](.github/actions/setup-rancher-cli/action.yml)             |
-| .travis.rancher_cli_kubernetes_config.yml | [setup-rancher-cli](.github/actions/setup-rancher-cli/action.yml)             |
-| .travis.srcclr_install.yml                | Not yet determined                                                            |
-| .travis.terraform-docs_install.yml        | [setup-terraform-docs](.github/actions/setup-terraform-docs/action.yml)       |
-| .travis.terraform_install.yml             | Preinstalled                                                                  |
-| .travis.tflint_install.yml                | Not yet determined                                                            |
-| .travis.trigger.yml                       | Not yet determined                                                            |
-| .travis.veracode.yml                      | [veracode](.github/actions/veracode)                                          |
-| .travis.yq_install.yml                    | Preinstalled                                                                  |
-
-### Default environment variables
-
-| Travis CI           | GitHub Actions          |
-|---------------------|-------------------------|
-| ${TRAVIS_BRANCH}    | ${{ github.ref_name }}  |
-| ${TRAVIS_BUILD_DIR} | ${{ github.workspace }} |
-| ${TRAVIS_COMMIT}    | ${{ github.sha }}       |
-
-### Get back maven build output
-
-Travis is very strict regarding maximum size of the build logs output and builds
-that exceed 5MB in output will fail with `The job exceeded the maximum log
-length, and has been terminated`. GitHub Actions doesn't have this kind of
-limitation and it's highly recommended to **remove any logs suppression** of the
-build tool.
-
-With Maven this is usually achieved by using the `-q` option, that is usually
-set globally inside the `MAVEN_CLI_OPTS` environment variable. Please remove any
-usage of `-q` when migrating from Travis.
-
-### Workflow schema validation
-
-The `.pre-commit-config.yaml` configuration should be updated to remove the obsolete `check-travis` hook and replace it with `check-github-workflows`.
-Note that a recent version of the `check-jsonschema` hook should be used to support reusable workflows.
-Here is a sample configuration:
-
-```yml
-  - repo: https://github.com/sirosen/check-jsonschema
-    rev: 0.17.0
-    hooks:
-      - id: check-github-workflows
-      - id: check-jsonschema
-        alias: check-dependabot
-        name: "Validate Dependabot Config"
-        files: '.github/dependabot\.yml$'
-        args: ["--schemafile", "https://json.schemastore.org/dependabot-2.0.json"]
-```
```

Comment:
It feels like this is worth documenting outside of the Travis migration doc, wdyt?

Reply:
we started to work on a new piece of doc specific to the pre-commit usage, will move this there once ready